### PR TITLE
Input min/max (and accidental indenting fix)

### PR DESCRIPTION
**Pull request by nkantar**
There could probably be better input validation based on the "too big"/"too many bombs" JS logic, but this is a start. :)

### CODE REVIEW

**Comment by muan** [comment]
Cooool. Can you change the quotes to double quote instead of single quotes though? Thank you!

**Comment by nkantar** [comment]
@muan Oops, sorry, totally spazzed and didn't follow the project's conventions. Fixed now.

File: index.html [hunk 1]
```diff
@@ -252,12 +252,12 @@
...
         <div class="flex-table">
-          <input id="cols" class="input flex-cell" type="number" value="10">
+          <input id="cols" class="input flex-cell" type="number" value="10" min="1" max="500">
           <div class="flex-cell-min divider">&times;</div>
-          <input id="rows" class="input flex-cell" type="number" value="10"><br>
+          <input id="rows" class="input flex-cell" type="number" value="10" min="1" max="500"><br>
         </div>
         <div class="prepend-input">💣</div>
-        <input id="bombs" class="input" type="number" value="10">
+        <input id="bombs" class="input" type="number" value="10" min='1' max='500'>
```
Comment (muan) [inline index.html]: Ooopppps also here. ;D

**Comment by muan** [comment]
✨✨✨✨ belated but thanks!

**Pull request merged**